sdfObject/sdfobject-onoff.sdf.json

```diff
@@ -111,16 +107,18 @@
           "type": "number", 
           "minimum": 0, 
           "maximum": 65535, 
-          "multipleOf": 1
+          "multipleOf": 1,
```

Comment:
We do have integer, wouldn't that be a more natural way to say this than the combination of number and multipleof=1?

Reply:
I made all of these cases "integer" to be consistent